app/assets/styles/styles.css

```diff
@@ -170,3 +170,137 @@ a {
     font-size: 1.25rem;
   }
 }
+
+.teaser {
+  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
+  font-size: 0.70rem;
+}
+
+@media only screen and (min-width: 700px) {
+  .teaser {
+   font-size: 0.85rem;
+  }
+}
+
+@media only screen and (min-width: 900px) {
+  .teaser {
+   font-size: 1rem;
+  }
+}
```

Comment:
I am not looking at the output because it isn't up on surge yet. However, if we apply the font change to the `teaser` which is the whole section element, I would assume that it is also going to affect the section title _Our News_. 

Do we want it to scale so small? 

In general, I find it easier to apply font changes per element, not at such a global level unless it clearly makes sense to because all the child elements are the same size. In this case, based on the designs, there are different font-sizes that we want to preserve.